x-pack/plugins/synthetics/public/apps/synthetics/hooks/use_monitor_enable_handler.tsx

```diff
@@ -41,11 +37,11 @@ export function useMonitorEnableHandler({
   const savedObjEnabledState = upsertStatuses[id]?.enabled;
   const [isEnabled, setIsEnabled] = useState<boolean | null>(null);
   const updateMonitorEnabledState = useCallback(
-    (monitor: EncryptedSyntheticsMonitor | MonitorOverviewItem, enabled: boolean) => {
+    (enabled: boolean) => {
       dispatch(
         fetchUpsertMonitorAction({
           id,
-          monitor: { ...monitor, [ConfigKey.ENABLED]: enabled },
+          monitor: { [ConfigKey.ENABLED]: enabled },
```

Comment:
Since edits are applied on top of the existing object, we can just pass the enabled key to be safe here.